src/jsd/egd_actuator.cc

```diff
@@ -133,6 +133,19 @@ void fastcat::EgdActuator::ElmoSetGainSchedulingIndex(uint16_t index)
   jsd_egd_set_gain_scheduling_index((jsd_t*)context_, slave_id_, true, index);
 }
 
+void fastcat::EgdActuator::ElmoFault()
+{
+  MSG("Faulting EGD through JSD: %s", name_.c_str());
+  jsd_egd_fault((jsd_t*)context_, slave_id_);
+
+  // TODO review with david
```

Comment:
I do not think the below assignments are necessary. When the Fastcat manager calls its `Process` function, the state from the underlying JSD device will be read first, so `jsd_egd_state_` will end up with the values assigned in `jsd_egd_fault`.
Only way this could not be the case would be if the JSD driver changed its rxpdo between the call to `jsd_egd_fault` and the call to `EgdActuator::PopulateState()`, and that is not possible because the writing and processing functions of the JSD driver are called AFTER the Fastcat manager reads states.